plugins/flytekit-k8s-pod/flytekitplugins/pod/task.py

```diff
@@ -19,6 +19,14 @@ def _sanitize_resource_name(resource: _task_models.Resources.ResourceEntry) -> s
 
 
 class Pod(object):
+    """
+    Pod is a platform-wide configuration that uses pod templates. Every default task is launched as a container in a pod.
```

Comment:
By default, every task is launched as a container in a pod. The pod itself is a platform-wide configuration that uses pod templates.